pkg/crc/version/version.go

```diff
@@ -47,6 +47,12 @@ type CrcReleaseInfo struct {
 	}
 }
 
+type VersionResult struct {
+	CrcVersion       string
+	OpenshiftVersion string
+	Success          bool
+}
+
```

Comment:
@anjannath doesn't we have all those struct as part of `pkg/crc/machine/types.go` ?

Reply:
@praveenkumar Yes, but this struct is actually not being used in the machine package or in the version package, it is used only in the `pkg/crc/api/handler.go` i'd prefer it to keep it in that file itself, but if you think this should be in the `pkg/crc/machine/types.go` i can move it there. I put it in `pkg/crc/version` package since its related to version.

Reply:
consolidation is important... all should be treated same.

Reply:
Moved it to `pkg/crc/machine/types.go`